src/phoenix/server/api/queries.py

```diff
@@ -78,6 +79,49 @@
 
 @strawberry.type
 class Query:
+    @strawberry.field
+    async def model_providers(
+        self, vendors: List[str], info: Info[Context, None]
+    ) -> List[ModelProvider]:
+        all_vendors = {
```

Comment:
Pythony question is there any reason to define this constant inside of a function instead of statically somewhere?

Reply:
Yeah it could be hoisted. But probably needs a refactor anyway